python/ql/lib/semmle/python/ApiGraphs.qll

```diff
@@ -485,6 +485,40 @@ module API {
       )
     }
 
+    /**
+     * Holds if `result` is the result of awaiting `awaitedValue`.
+     */
+    cached
+    DataFlow::Node awaited(DataFlow::Node awaitedValue) {
```

Comment:
As pointed out elsewhere, I'm wondering if there is any reason it's not called `getAwaited`? -- such that it makes our QL style guide for predicates with results.